assets/css/common/mixins.scss

```diff
@@ -40,6 +40,20 @@ $is-ie: false !default;
   box-shadow:         $shadow;
 }
 
+@mixin box-sizing($sizing) {
+  -webkit-box-sizing: $sizing;
+  -moz-box-sizing:    $sizing;
+  box-sizing:         $sizing;
+}
+
+@mixin transform($transform) {
+  -moz-transform: $transform; // Firefox 15 compatibility
+  -o-transform: $transform; // Opera 12.0 compatibility
+  -webkit-transform: $transform; // Safari 8 compatibility
+  -ms-transform: $transform; // IE9 compatibility
```

Comment:
We don't worry about any of those browsers: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices